plugins/janus_videoroom.c

```diff
@@ -441,6 +442,11 @@ typedef struct janus_videoroom_rtp_relay_packet {
 
 
 /* Freeing stuff */
+static void janus_videoroom_subscriber_dereference(janus_videoroom_subscriber *s) {
+	if(s)
```

Comment:
Oh God please no, no asserts... I hate it enough when libraries decide to assert and abort my application without giving me the chance to handle the error case. Not sure about removing those checks, they seem quick and harmless enough.